BUILD.bazel

```diff
@@ -2679,33 +2679,28 @@ ray_cc_test(
 )
 
 ray_cc_library(
-    name = "gcs",
+    name = "gcs_pb_util",
     srcs = [
         "//:src/ray/gcs/pb_utils.cc",
-        "//:src/ray/gcs/redis_async_context.cc",
-        "//:src/ray/gcs/redis_client.cc",
-        "//:src/ray/gcs/redis_context.cc",
```

Comment:
These is an existed target to build these files.
https://github.com/ray-project/ray/blob/4ff0739be73c478151255522f0ff61be167afbb9/BUILD.bazel#L2520-L2525

Reply:
That's the whole purpose for the ticket! Thank you so much!